src/DotnetWebApiBench.DataAccess/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -33,13 +33,18 @@ public static class ServiceCollectionExtensions
     {
         private static DbConnection dbConnection;
         public static void AddNorthwindDataAccess(this IServiceCollection services, 
-            string connectionString)
+            string connectionString, string serverType)
```

Comment:
Instead of passing database type as a string I suggest to create an Enum with all supported database types (currently it would contain `SQLite` and `SQLServer` values). Applies to all occurrences of the serverType string.

Reply:
changes applied